bin/compile

```diff
@@ -22,5 +22,10 @@ echo "-----> cloning sphinx document project"
 cd $BUILD_DIR
 git clone $GIT_URL repo
 
+echo "-----> installing 3rd-party python packages"
+if [ -r repo/requirements.txt ]; then
+  pip install -r repo/requirements.txt
+fi
```

Comment:
Can you please update `limitation` https://github.com/shimizukawa/heroku-sphinxbuild-template/blame/master/README.rst#L25

and ChangeLog https://github.com/shimizukawa/heroku-sphinxbuild-template/blob/master/ChangeLog.rst

Reply:
done